chapters/modules/chapter_2/pages/calling_invoking.adoc

```diff
@@ -4,9 +4,111 @@
 
 Let's interact with our deployed voting contract.
 
-* Calling contracts refer to interacting with contract functions with the `@view` entry point (decorator). Since we are not altering the state of the network, and hence we do not need to pay fees, we do not need to sign the operation.
+* Calling contracts refer to interacting with contract functions with the `@view` entry point (decorator). Since we are not altering the state of the network, we do not need to pay fees and we do not need to sign the operation.
 * Invoking contracts is interacting with contract functions with the `@external` entry point. We need to sign the transaction and pay the operation fee.
 
+Here is the command that we used, to deploy the contract. Note that the voter accounts registered during deployment are the only ones that can vote.
```

Comment:
I would make explicit where these accounts are specified. Either write a generic command with <account1> <account2> etc, or write the account in the description. 
I prefer the former though.